selfdrive/car/subaru/values.py

```diff
@@ -53,8 +53,13 @@ def __init__(self, CP):
 
 
 class SubaruFlags(IntFlag):
-  SEND_INFOTAINMENT = 1
```

Comment:
it's useful to keep these static so that we can easily determine the flags from the logs